examples/weather_skills/README.md

```diff
@@ -53,10 +62,16 @@ ledger_apis:
     addr: alpha.fetch-ai.com
     port: 80
 ```
+- Generate the private key for the weather client:
+
+     aea run
```

Comment:
mention the `-v DEBUG` flag at some point? So we remind the user (or make him aware) of this option.

Reply:
No I think we have this in the main docs. Let's keep things simple and brief.